### PR TITLE
AndrewTehan/hometask-5

### PR DESCRIPTION
**Pull request by AndrewTehan**
[hometask-5](https://docs.google.com/document/d/1DyuPaRI2wiw-bCOg3Ht0XbQsLr_PZnIR2vIr0O8nfNU/edit)

### CODE REVIEW

**Comment by AndrewTehan** [comment]
Codecademy SQL course is completed
image

File: AndrewTehan/hometask-5/4_2.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+select * from film
```
Comment (hannazspot) [inline AndrewTehan/hometask-5/4_2.sql]: films and languages are connected with 1-1 relationship, so `AND` doesn't suit there

File: AndrewTehan/hometask-5/4_3.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+select concat(first_name, '-', last_name) as fullname from actor
```
Comment (hannazspot) [inline AndrewTehan/hometask-5/4_3.sql]: Screen Shot 2021-11-21 at 8 28 42 PM

File: AndrewTehan/hometask-5/4_7.sql [hunk 3]
```diff
@@ -0,0 +1,4 @@
+update film
+  set description = 'Dune tells the story of Paul Atreides, a brilliant and gifted young man born into a great destiny beyond his understanding, who must travel to the most dangerous
```
Comment (hannazspot) [inline AndrewTehan/hometask-5/4_7.sql]: Screen Shot 2021-11-21 at 8 33 19 PM

**Review by hannazspot — CHANGES_REQUESTED** [review]

File: AndrewTehan/hometask-5/4_2.sql [hunk 4]
```diff
@@ -0,0 +1,7 @@
+select title, l.name 
+from film f
+  inner join language l 
+    on f.language_id = l.language_id 
+where l.name = 'English' 
+  or l.name = 'German' 
+  or l.name = 'Italian';
```
Comment (hannazspot) [inline AndrewTehan/hometask-5/4_2.sql]: instead of such amount of `OR` operators, you may use `IN`

**Review by hannazspot — APPROVED** [review]

**Pull request merged**